.eslintrc.js

```diff
@@ -23,7 +23,7 @@ module.exports = {
     ecmaFeatures: {
       jsx: true,
     },
-    project: ['./tsconfig.json', './cypress/tsconfig.json'],
+    project: ['./tsconfig.eslint.json', './cypress/tsconfig.json'],
```

Comment:
Specific eslint config because we ignore `spec` and `stories` directories for Typescript, but want to cover them with linting... Perhaps we'll revisit that when we move our build steps to using Babel instead of Typescript. That way we can type check all Typescript files.